exotations/solvers/ompl_imp_solver/src/ompl_imp_solver/OMPLBaseStateSpace.cpp

```diff
@@ -70,7 +70,7 @@ namespace exotica
         state, q);
 #endif
 
-    if (!prob_->isValid(q) || !prob_->getScene()->getCollisionScene()->isStateValid(self_collision_, margin_))
+    if (!prob_->isValid(q) || !prob_->getScene()->getCollisionScene()->isStateValid(self_collision_))
```

Comment:
The margin affects the broad phase solver. It would require a new broad phase solver implementation to add this. This only reduces the number of required broad phase checks though. The current solver will still return a correct result. Possibly with a bit of extra work.

Reply:
We should definitely keep this in mind and on the list of required updates as our sampling based motion planning currently relies on this. Alternatively we can add link and world padding which MoveIt supports, not sure how easy "inflation" would be with our own FCL Scene.